source/common/filter/config_discovery_impl.h

```diff
@@ -29,6 +29,24 @@ class FilterConfigSubscription;
 
 using FilterConfigSubscriptionSharedPtr = std::shared_ptr<FilterConfigSubscription>;
 
+enum STAT_PREFIX_TYPE { HTTP_FILTER, TCP_LISTENER_FILTER, UDP_LISTENER_FILTER, MAX_FILTER_INDEX };
```

Comment:
Nit: Please use Envoy style `StatPrefixType`, `HttpFilter` etc.

Reply:
Thanks. This enum is now removed. We changed into directly using reference of StatPrefix strinig for non-typed template parameter in
emplate <class Factory, class FactoryCb, class FactoryCtx, class ListenerDynamicFilterConfig,
          const absl::string_view& StatPrefix>
class FilterConfigProviderManagerImpl